src/styles/calendar.scss

```diff
@@ -0,0 +1,55 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+  gap: $gap;
+  padding: 10px;
+  margin: 0 auto;
+
+  &__day {
+    box-sizing: border-box;
+    height: $day-size;
+    width: $day-size;
+    border: 1px solid #000;
+    background-color: #eee;
+    position: relative;
+    transition:
+      transform 0,
+      5s,
+      background-color 0.5s;
+
+    @for $i from 1 through $days-in-month {
+      &:nth-child(#{$i})::before {
+        content: '#{$i}';
+        position: absolute;
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: 30px;
+        color: #000;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
```

Comment:
These styles are the same for all elements, move them outside the loop. And it's better to center with display flex instead of position absolute